app/src/main/java/com/keylesspalace/tusky/db/AccountManager.kt

```diff
@@ -243,4 +267,14 @@ class AccountManager @Inject constructor(db: AppDatabase) {
 
         return accounts.size > 1 // "disambiguate"
     }
+
+    /**
+     * Clear drafts alert for current user
+     * Caller's responsibility to ensure there is a current user
+     */
+    fun clearDraftsAlert() {
+        runBlocking {
+            draftDao.draftsClearNeedUserAlert(activeAccount!!.id)
+        }
```

Comment:
Runblocking might be bad here.

Also, is there a race condition here with activeAccount? IE could we get a post fail notification on one user then somehow the active user changes between draftsNeedUserAlert deciding to signal its observer and the call back to clearDraftsAlert occurring?

Reply:
If there is, could you fix that by passing the ID in as a parameter to the function instead?

Reply:
I think I could do this in the mediator callback, yes, but I do not know if there is a race condition.